python/ray/tune/tests/test_integration_wandb.py

```diff
@@ -50,23 +53,39 @@ def __str__(self):
         return self.trial_name
 
 
+@dataclass
+class _MockWandbConfig:
+    args: Tuple
+    kwargs: Dict
+
+
+class _MockWandbAPI(object):
```

Comment:
```suggestion
class _MockWandbAPI:
```

We don't support Python 2, right?

Reply:
done